apps/other/traccar.md

```diff
@@ -0,0 +1,199 @@
+# Traccar
+
+![traccar_hu4f4d748acb69c0f52e2340aac180cda5_35275_1600x0_resize_box_3](https://user-images.githubusercontent.com/52239579/212997283-e3279b61-189f-452c-86e5-d1e2fe216da2.png)
+
+**Traccar is a gps tracking system. You can use it to monitor your smartphone location history**
+
+<br>
+
+[website](https://www.traccar.org/)
+
+[documentation](https://www.traccar.org/documentation/)
+
+## get the default traccar conf 
+
+`docker run --rm --entrypoint cat debian-traccar-nginx:latest /opt/traccar/conf/traccar.xml > /run/media/ippo/TOSHIBA/traccar/conf/traccar.xml`
+
+## Configuration parameters for MySQL 
+(replace [DATABASE], [USER], [PASSWORD] with appropriate values from docker-compose replace [HOST "IPv4Address" from db_name section in docker network inspect)
+
+```
+<entry key='database.driver'>com.mysql.cj.jdbc.Driver</entry>
+<entry key='database.url'>jdbc:mysql://[HOST]:3306/[DATABASE]?serverTimezone=UTC&amp;allowPublicKeyRetrieval=true&amp;useSSL=false&amp;allowMultiQueries=true&amp;autoReconnect=true&amp;useUnicode=yes&amp;characterEncoding=UTF-8&amp;sessionVariables=sql_mode=''</entry>
+<entry key='database.user'>[USER]</entry>
+<entry key='database.password'>[PASSWORD]</entry>
+```
```

Comment:
what file is this? where would one put this xml?

Reply:
Traccar uses a configuration file for varius parameters
https://www.traccar.org/configuration-file/
One of this is the dtabase
By dwfault it uses an embeded  h2 db good for teating but not suited for production
An external mysql or postgres db is needed for production
The mysql cofiguretion parameters are here
 https://www.traccar.org/mysql/

Reply:
ok. understood, thanks!